pkg/virt-launcher/virtwrap/cmd-server/server_test.go

```diff
@@ -375,7 +375,17 @@ var _ = Describe("Virt remote commands", func() {
 				Expect(err).ToNot(HaveOccurred())
 				Expect(resp.Response.Success).To(BeTrue())
 			})
+			It("should get the qemu version", func() {
```

Comment:
Could have left this in the server context, it think it fits better there?

Reply:
Done!